ParisBNF/abb/BNFabb3.xml

```diff
@@ -0,0 +1,327 @@
+                       <date>1902</date>.
+                    </acquisition>
+               </history>
+
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>
+                           <listBibl type="catalogue">
+                              <listBibl type="catalogue">
+                                 <bibl>
+                                    <ptr target="bm:ContiRossini1914Notice"/>
+                                    <citedRange unit="page">48-49</citedRange>
+                                    <citedRange unit="number">20</citedRange>
+                                    </bibl>
+                                 <bibl>
+                                    <ptr target="bm:Chaine1912Catalogue"/>
+                                    <citedRange unit="page">2</citedRange>
+                                    <citedRange unit="number">3</citedRange>
+                                 </bibl>
+                                 <bibl>
+                                    <ptr target="bm:Abbadie1859Cat"/>
+                                    <citedRange unit="page">2-3</citedRange>
+                                    <citedRange unit="number">3</citedRange>
+                                 </bibl>
+                                   </listBibl>
+
+                           </listBibl>
+                        </source>
+                     </recordHist>
+                  </adminInfo>
+               </additional>
+            </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <particDesc>
+            <listPerson>
+              <person>
+                 <persName ref="PRS1071dAbbadi" role="bequeather"/>
+              </person>
+              <person>
+                 <persName ref="PRS1071dAbbadi" role="owner"/>
+                 </person>
+                 <person>
+                    <persName ref="PRS13325Adilu" role="owner" cert="low"/>
+                    </person>
+            </listPerson>
+         </particDesc>
+         <textClass>
+            <keywords scheme="#ethioauthlist">
+               <term key="OldTestament"/>
+                <term key="Prayers"/>
+               <term key="ChristianLiterature"/>
+               <term key="Gon"/>
+               <term key="ZaMa"/>
+            </keywords>
+         </textClass>
+         <langUsage><language ident="gez">Gǝʿǝz</language><language ident="la">Latin</language></langUsage>
```

Comment:
add am

Reply:
and fr

Reply:
ok

Reply:
and tn if you mark up tigrinya words too